library.js

```diff
@@ -255,7 +255,7 @@ async function generateBody(req, postData) {
 		return `${translated}\n` +
 			`> ${postData ? `${postData.content.replace(/\n/g, '\n> ')}\n\n` : ''}`;
 	} else if (req.query.body) {
-		return req.query.body;
+		return validator.escape(String(req.query.body));
```

Comment:
This is explicitly **not** a zero-day. Despite it not having been sanitized before today, `body` was never actually output to the template, so it was not possible to exploit.